crates/cli-support/src/js/mod.rs

```diff
@@ -1065,58 +1065,32 @@ impl<'a> Context<'a> {
         if !self.exposed_globals.insert("text_encoder") {
             return;
         }
-        if self.config.nodejs_experimental_modules {
-            self.imports
-                .push_str("import { TextEncoder } from 'util';\n");
-        } else if self.config.nodejs {
-            self.global(
-                "
-                const TextEncoder = require('util').TextEncoder;
-                ",
-            );
-        } else if !(self.config.browser || self.config.no_modules) {
-            self.global(
-                "
-                const TextEncoder = typeof self === 'object' && self.TextEncoder
-                    ? self.TextEncoder
-                    : require('util').TextEncoder;
-                ",
-            );
-        }
-        self.global(
-            "
-            let cachedEncoder = new TextEncoder('utf-8');
-            ",
-        );
+        self.expose_text_processor("TextEncoder");
     }
 
     fn expose_text_decoder(&mut self) {
         if !self.exposed_globals.insert("text_decoder") {
             return;
         }
+        self.expose_text_processor("TextDecoder");
+    }
+
+    fn expose_text_processor(&mut self, s: &str) {
         if self.config.nodejs_experimental_modules {
-            self.imports
-                .push_str("import { TextDecoder } from 'util';\n");
+            self.imports.push_str(&format!("impot {{ {} }} from 'util';\n", s));
```

Comment:
I assume this should be `import ...`?

Reply:
Oops yes indeed! Thanks for testing regardless!